### ISSUE TITLE
Issue with TMUX

### ISSUE THREAD

**Issue by juimdpp** [swsnu/osspr2021#66]
안녕하세요 조교님!
과제를 수행하는 도중 tmux가 안 깔려서 문제를 겪고 있어 어려움을 느끼고 있습니다. 
지시대로 tmux tar 파일을 다운 받고, install-tmux.sh을 실행할 때 다음과 같은 메시지가 뜹니다:
<img width="717" alt="Screen Shot 2021-04-09 at 21 19 13" src="https://user-images.githubusercontent.com/60267222/114178873-3d17e200-9979-11eb-8f03-9bf3e5baee18.png">
어떻게 해결해야할지 몰라서 물어봅니다!
도와주시면 큰 도움이 될것 같습니다. 
감사합니다.

**Comment by jaywonchung** [comment]
Check the output of `mount`. You'll probably have an image mounted to a temporary directory (e.g. `/tmp/tmp.xxxxxxx`). Run `sudo umount` on that directory and the problem should go away.

**Comment by juimdpp** [comment]
와... 감사합니다 ㅠㅠ

**Comment by Kangji** [comment]
Hi, I am having a trouble with the same issue. I did `sudo umount /tmp/*`, but still same. Is there any other ideas to solve this? Thnx.

**Comment by Kangji** [comment]
Lol I just restarted my computer and now it works!

**Comment by jaywonchung** [comment]
Yup, that's the ultimate solution. Regardless of the detail it's a mount problem, and *all* mounts that are not recorded in `/etc/fstab` gets removed when the system reboots :)
